app/src/main/java/io/github/droidkaigi/confsched/activity/MainActivity.java

```diff
@@ -132,10 +156,13 @@ private void toggleToolbarElevation(boolean enable) {
         }
     }
 
-    private void changePage(@StringRes int titleRes, @NonNull Fragment fragment) {
+    private void changePage(Page page) {
+        binding.appbar.setExpanded(true, true);
```

Comment:
If you are in MapFragment ,You can't expand AppBarLayout. 
So I expanded AppBarLayout programmatically on changePage.